src/Event.c

```diff
@@ -7,18 +7,20 @@
 
 struct Event 
 {
-    void (*fun_ptr_arr[MAX_LISTNERS])(void); // 8*16 = 128
-    uint8_t currentRegisteredListeners; // 8
+    void (*fun_ptr_arr[MAX_LISTNERS])(void); 
+    uint8_t currentRegisteredListeners;
 };
 
 #define MAX_EVENTS 32
 
-static struct Event mEventStorage[MAX_EVENTS]; // 136 * 32 = 4,352 ?
-static EventPtr mEventPointers[MAX_EVENTS]; // 8*32
+static struct Event mEventStorage[MAX_EVENTS];
+static EventPtr mEventPointers[MAX_EVENTS];
 static uint8_t mNextUnusedEvent;
 
-// allocate some space to store deleted
-static size_t pointerStorage;
+static bool Event_internal_getElementIndex(EventPtr event, void (*ptr)(void), int* index);
+static void Event_internal_wipeAllListeners(EventPtr event);
```

Comment:
The convention we typically use is `Event_` implies that it's part of the public interface, and anything without that is an internal helper function. So just `static bool getElementIndex(EventPtr event, void (*ptr)(void), int* index);`